src/abonne.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Ipssi\Evaluation;
+
+class abonne {
+
+    private $numeroAuto = 0;
+
+    private $numero;
+
+    private $estInterditEmprunt;
+
+    private $dateFinInterdit;
+
+
+
+    /**
+     * abonne constructor.
+     * @param int $numeroAuto
+     * @param $numero
+     * @param $estInterditEmprunt
+     * @param $dateFinInterdit
+     */
+    public function __construct(int $numeroAuto, $numero, $estInterditEmprunt)
```

Comment:
```suggestion
    public function __construct(int $numeroAuto)
```
Vous ne les utilisez pas, donc pas utile comme arguments